src/machine.rs

```diff
@@ -54,15 +55,23 @@ pub struct FrameData<'tcx> {
     /// for the start of this frame. When we finish executing this frame,
     /// we use this to register a completed event with `measureme`.
     pub timing: Option<measureme::DetachedTiming>,
+
+    /// If this frame is protecting any tags, they are listed here. We use this list to do
+    /// incremental updates of the global list of protected tags stored in the
+    /// `stacked_borrows::GlobalState` upon function return, and if we attempt to pop a protected
+    /// tag, to identify which call is responsible for protecting the tag.
+    /// See `Stack::item_popped` for more explanation.
+    pub protected_tags: SmallVec<[SbTag; 2]>,
```

Comment:
I had a request somewhere to please only have one field per "subsystem" in `FrameData`. Looks like that discussion got resolved without actually completing the change. So please put the two Stacked Borrows field into a new type defined inside the Stacked Borrows module.

Reply:
Sorry if I accidentally marked that as resolved, I definitely have no memory of this comment.

I'm not really sure of the naming here. There's not really any particular concept that ties the CallId and protected tags together other than that they are for Stacked Borrows.

Reply:
I might also have failed to actually send it, or github lost it. :shrug: 

The name looks good!